profile/README.md

```diff
@@ -41,5 +41,12 @@ July 24th, 2023 | Edinburgh, Scotland | [Agile Scotland](https://www.agileallian
 August 2nd, 2023 | Virtual | [SmartBear Webinar](https://smartbear.com/resources/webinars/) | [API Security by Design](https://smartbear.com/resources/webinars/api-security-by-design/) | Frank Kilcommins, Jose Haro Peralta | [Recording](https://www.youtube.com/watch?v=acXpD1tRmCQ) [Slides](https://assets.smartbear.com/transfer/388708a6e1980ed76ed7a194110d44f95c49790b294da03aa5906aea28806ce1)
 Sept 14th, 2023 | London, England | [apidays](https://www.apidays.global/london/) | [The API Workflows Specification - the missing piece of the API puzzle!](https://www.openapis.org/events/apidays-london-2023) | Frank Kilcommins | [Slides](https://noti.st/frankkilcommins/FfbA40/the-workflows-specification-the-missing-piece-of-the-api-puzzle)
 Sept 28th, 2023 | Leeds, England | [Leeds Digital Festival](https://leedsdigitalfestival.org/) | [Engineering Excellence through Testing](https://www.meetup.com/transcend-tech-leeds/events/295886706) | Yousaf Nabi |
+Oct 10th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Improving Software Quality with Contract Testing and Pact](https://www.youtube.com/watch?v=5vAV1lTGJyE) | Seb Rose | [YouTube](https://www.youtube.com/watch?v=5vAV1lTGJyE) |
+Oct 10th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pactober 2023 - Pact Workshop (HTTP)](https://www.youtube.com/watch?v=Gx-R2Cn1HZE) | Yousaf Nabi | [YouTube](https://www.youtube.com/watch?v=Gx-R2Cn1HZE) |
+Oct 10th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pacts 10th Birthday Bash](https://www.youtube.com/watch?v=s1LOP4Ak5SA) | Various Speakers | [YouTube](https://www.youtube.com/watch?v=s1LOP4Ak5SA)
+Oct 11th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pactober 2023 - Message Pact](https://www.youtube.com/watch?v=81N25-4x27E) | Yousaf Nabi | [YouTube](https://www.youtube.com/watch?v=81N25-4x27E) |
+Oct 17th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pactober 2023 - Pact Plugins](https://www.youtube.com/watch?v=0FpzPRSf2VA) | Yousaf Nabi | [YouTube](https://www.youtube.com/watch?v=0FpzPRSf2VA) |
+**Upcoming** Oct 24th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pactober 2023 - Pact Maintainer Workshop](https://www.youtube.com/watch?v=QIza-D-f7DI) | Yousaf Nabi | [YouTube](https://www.youtube.com/watch?v=QIza-D-f7DI) | 
+**Upcoming** Oct 27th, 2023 | Virtual | [Pactober - 2023](https://pact.io/pactober) | [Pactober - PactBuilderDSL - presented by Sngular](https://www.youtube.com/watch?v=whXaNti7PSg) | Alejandro Pena / José Maciñeiras | [YouTube](https://www.youtube.com/watch?v=whXaNti7PSg) |
```

Comment:
Whoop - thanks for this! Could you just move this one below the security webinar, so that all are ordered by date? Cheers